RestaurantsNearMe/Mocks/AutoGeneratedMocks.swift

```diff
@@ -128,11 +128,16 @@ class LocationProvidingMock: LocationProviding {
 class RestaurantListProvingMock: RestaurantListProving {
 
 
-    var restaurants: PassthroughSubject<[Restaurant], Never> {
+    var restaurants: AnyPublisher<[Restaurant], Never> {
         get { return underlyingRestaurants }
         set(value) { underlyingRestaurants = value }
     }
-    var underlyingRestaurants: (PassthroughSubject<[Restaurant], Never>)!
+    var underlyingRestaurants: (AnyPublisher<[Restaurant], Never>)!
```

Comment:
Coming from `RxSwift` at least in the previous codebase I was using almost all `streams` of data we're infinite (ex:
streams of camera records coming in at 30 FPS) If any error occurred within the stream, the stream itself would continue to receive events, and the errors would be handled separately. 

In cases like this, is it more common that stop the stream and create a new one when an error occurs (i.e replacing `Never` with some Error type and restart it based on the next call) Or is this setup okay?